sieve.py

```diff
@@ -119,6 +119,10 @@ def setup_cluster(project, stage, mode, test_config, docker_repo, docker_tag, nu
         os.system("docker pull %s" % (image))
         os.system(kind_load_cmd)
 
+    if num_workers == 0:
```

Comment:
Let's make the csi-driver an option in the test suite and enable it for the rabbitmq pvc resize only, instead of checking the number of workers